portfolio/src/main/webapp/index.html

```diff
@@ -68,11 +76,80 @@ <h2 class="bios opener" id="computabeast"> AKA: THE computabeast </h2>
             </div>
         </div>
 
-        <div class="part">
+        <div class="part" id="secondpart">
+            <div id="links-table">
+                <table class="tabler">
+                        <td><img id="coolgif" src="images/5eKX.gif" alt="animated" /></td>
+                        <!-- https://gifer.com/en/5eKX -->
+                </table>
+            </div>
+
+             <div class="text-container" id="education">
+                <h1>Education</h1>
+
+                <table class="centerTable">
+                    <tr>
+                        <h2>Yale University - B. S. in Computer Science*</h2>
+                    </tr>
+                    <tr>
+                        <p>*anticipated May 2022</p>
+                    </tr>
+                    <tr>
+                        <p>Classes taken include:</p>
+                    </tr>
+                    <tr>
+                        <td><h3>CPSC 202 Mathematical Tools for Computer Science</h3></td>
+                        <td><h3>CPSC 223 Data Structures and Programming Techniques (Spring 2018)</h3></td>
+                    </tr>
+                    <tr>
+                        <td><h3>CPSC 437 Introduction to Database Systems (Fall 2019)</h3></td>
+                        <td><h3>CPSC 323 Introduction to Systems Programming and Computer Organization (Spring 2020)</h3></td>
+                    </tr>
+                </table>
+            </div>
+        </div>
+
+        <div class="part" id="thirdpart">
+
+             <div class="text-container">
+                <h1>Projects</h1>
+
+                <p class="bios" id="projects1" > Here are some of the projects that I've done:</p>
+
+                <dl class="bios" id="projects2">
+                    <dt class="dts"><a href="http://2wocents.com">2wocents.com</a></dt>
+                        <dd>- Social network I created to hone my computer science skills</dd>
+                    <dt class="dts"><a href="https://apps.apple.com/us/app/mr-roll/id1103794172">Mr. Roll</a></dt>
+                        <dd>- Simple run and jump iPhone game to have some fun with my friends</dd>
+                    <dt class="dts">Raspberry Pi Bluetooth Beacon</dt>
+                        <dd>- I configured a RasPi to emit a bluetooth signal upon startup, which is read 
```

Comment:
Inner HTML that's multiple lines should be on the next line and indented. This has been a repeat comment in your PR's. Please take care to address this before sending future reviews.

Reply:
Please fix everywhere in this file.

Reply:
Done. Sorry about that